apps/condo/migrations/20231124183332-0345_remove_billingaccount_billingaccount_unique_context_globalid_and_more.js

```diff
@@ -0,0 +1,90 @@
+// auto generated by kmigrator
+// KMIGRATOR:0345_remove_billingaccount_billingaccount_unique_context_globalid_and_more:IyBHZW5lcmF0ZWQgYnkgRGphbmdvIDQuMi41IG9uIDIwMjMtMTEtMjQgMTM6MzMKCmZyb20gZGphbmdvLmRiIGltcG9ydCBtaWdyYXRpb25zLCBtb2RlbHMKCgpjbGFzcyBNaWdyYXRpb24obWlncmF0aW9ucy5NaWdyYXRpb24pOgoKICAgIGRlcGVuZGVuY2llcyA9IFsKICAgICAgICAoJ19kamFuZ29fc2NoZW1hJywgJzAzNDRfYjJjYXBwX2ltcG9ydGlkX2IyY2FwcF9pbXBvcnRyZW1vdGVzeXN0ZW1fYW5kX21vcmUnKSwKICAgIF0KCiAgICBvcGVyYXRpb25zID0gWwogICAgICAgIG1pZ3JhdGlvbnMuUmVtb3ZlQ29uc3RyYWludCgKICAgICAgICAgICAgbW9kZWxfbmFtZT0nYmlsbGluZ2FjY291bnQnLAogICAgICAgICAgICBuYW1lPSdiaWxsaW5nQWNjb3VudF91bmlxdWVfY29udGV4dF9nbG9iYWxJZCcsCiAgICAgICAgKSwKICAgICAgICBtaWdyYXRpb25zLlJlbW92ZUNvbnN0cmFpbnQoCiAgICAgICAgICAgIG1vZGVsX25hbWU9J2JpbGxpbmdwcm9wZXJ0eScsCiAgICAgICAgICAgIG5hbWU9J2JpbGxpbmdQcm9wZXJ0eV91bmlxdWVfY29udGV4dF9nbG9iYWxJZCcsCiAgICAgICAgKSwKICAgICAgICBtaWdyYXRpb25zLkFsdGVyRmllbGQoCiAgICAgICAgICAgIG1vZGVsX25hbWU9J2JpbGxpbmdhY2NvdW50JywKICAgICAgICAgICAgbmFtZT0nbWV0YScsCiAgICAgICAgICAgIGZpZWxkPW1vZGVscy5KU09ORmllbGQoYmxhbms9VHJ1ZSwgbnVsbD1UcnVlKSwKICAgICAgICApLAogICAgICAgIG1pZ3JhdGlvbnMuQWx0ZXJGaWVsZCgKICAgICAgICAgICAgbW9kZWxfbmFtZT0nYmlsbGluZ2FjY291bnQnLAogICAgICAgICAgICBuYW1lPSdyYXcnLAogICAgICAgICAgICBmaWVsZD1tb2RlbHMuSlNPTkZpZWxkKGJsYW5rPVRydWUsIG51bGw9VHJ1ZSksCiAgICAgICAgKSwKICAgICAgICBtaWdyYXRpb25zLkFsdGVyRmllbGQoCiAgICAgICAgICAgIG1vZGVsX25hbWU9J2JpbGxpbmdwcm9wZXJ0eScsCiAgICAgICAgICAgIG5hbWU9J2dsb2JhbElkJywKICAgICAgICAgICAgZmllbGQ9bW9kZWxzLlRleHRGaWVsZChibGFuaz1UcnVlLCBudWxsPVRydWUpLAogICAgICAgICksCiAgICAgICAgbWlncmF0aW9ucy5BbHRlckZpZWxkKAogICAgICAgICAgICBtb2RlbF9uYW1lPSdiaWxsaW5ncHJvcGVydHknLAogICAgICAgICAgICBuYW1lPSdtZXRhJywKICAgICAgICAgICAgZmllbGQ9bW9kZWxzLkpTT05GaWVsZChibGFuaz1UcnVlLCBudWxsPVRydWUpLAogICAgICAgICksCiAgICAgICAgbWlncmF0aW9ucy5BbHRlckZpZWxkKAogICAgICAgICAgICBtb2RlbF9uYW1lPSdiaWxsaW5ncHJvcGVydHknLAogICAgICAgICAgICBuYW1lPSdyYXcnLAogICAgICAgICAgICBmaWVsZD1tb2RlbHMuSlNPTkZpZWxkKGJsYW5rPVRydWUsIG51bGw9VHJ1ZSksCiAgICAgICAgKSwKICAgICAgICBtaWdyYXRpb25zLkFsdGVyRmllbGQoCiAgICAgICAgICAgIG1vZGVsX25hbWU9J2JpbGxpbmdyZWNlaXB0JywKICAgICAgICAgICAgbmFtZT0ncmF3JywKICAgICAgICAgICAgZmllbGQ9bW9kZWxzLkpTT05GaWVsZChibGFuaz1UcnVlLCBudWxsPVRydWUpLAogICAgICAgICksCiAgICAgICAgbWlncmF0aW9ucy5BbHRlckZpZWxkKAogICAgICAgICAgICBtb2RlbF9uYW1lPSdiaWxsaW5ncmVjZWlwdCcsCiAgICAgICAgICAgIG5hbWU9J3JlY2lwaWVudCcsCiAgICAgICAgICAgIGZpZWxkPW1vZGVscy5KU09ORmllbGQoYmxhbms9VHJ1ZSwgbnVsbD1UcnVlKSwKICAgICAgICApLAogICAgXQo=
+
+exports.up = async (knex) => {
```

Comment:
I think this migration needs 120+ seconds to complete. Lets remove timeout

Reply:
Done